src/components/About/Blurbs.js

```diff
@@ -75,16 +85,18 @@ class Blurbs extends Component {
 
         <div className = "blurbDivs">
 
-          <h3 className = "endingBlurbs">
-            And that's the story of how fortissimo was conceived.
-          </h3>
-
           <img className = "fillRightBlurbImages" src={require("../../img/img-logo-colored.jpg")} alt="fortissimo-logo"/ >
 
+          <div className = "BlurbContainer">
+            <h3 className = "BlurbText">
+              And that's the story of how fortissimo was conceived.
+            </h3>
+          </div>
+
         </div>
       </div>
     );
   }
 }
 
-export default Blurbs;
+export default BlurbContainer;
```

Comment:
About/Blurbs.js: Code is clean, good use of `<div className = "BlurbContainer">`  and `<h3 className = "BlurbText">`  to align the blurbs and text, and eases future additions/amendments.